models.js

```diff
@@ -0,0 +1,15 @@
+const db = require('./db/connection');
+const fs = require('fs/promises')
+const endpoints = require('./endpoints.json')
+
+
+
+exports.getTopics = () => {
+    return db.query('SELECT * FROM topics').then((results) => {
+        return results.rows
+    })
+}
+
+exports.getEndpoints = () => {
```

Comment:
we don't need a model for the endpoints, lets pop this functionality in the controller!